README.rst

```diff
@@ -55,189 +55,45 @@ Installation
 
 Concepts
 ========
-There are four main concepts that you need to understand to use this library:
+There are three main concepts that are helpful to understand when using the AWS Encryption SDK.
+
+For further information, see the `AWS Encryption SDK developer guide concepts`_.
 
 Cryptographic Materials Managers
 --------------------------------
-Cryptographic materials managers (CMMs) are resources that collect cryptographic materials and prepare them for
-use by the Encryption SDK core logic.
-
-An example of a CMM is the default CMM, which is automatically generated anywhere a caller provides a master
-key provider. The default CMM collects encrypted data keys from all master keys referenced by the master key
-provider.
-
-An example of a more advanced CMM is the caching CMM, which caches cryptographic materials provided by another CMM.
+The cryptographic materials manager (CMM) assembles the cryptographic materials
+that are used to encrypt and decrypt data.
+The cryptographic materials include plaintext and encrypted data keys, and an optional message signing key.
+You can use the default CMM that the AWS Encryption SDK provides or write a custom CMM.
+You can specify a CMM, but you never interact with it directly.
+The encryption and decryption methods handle it for you.
 
-Master Key Providers
---------------------
-Master key providers are resources that provide master keys.
-An example of a master key provider is `AWS KMS`_.
+The default CMM gets the encryption or decryption materials from
+the keyring or master key provider that you specify.
+This might involve a call to a cryptographic service, such as AWS Key Management Service (AWS KMS).
```

Comment:
This and the descriptions for keyrings and data keys are just copied from the developer guide concepts docs. Thinking this over, to avoid needing to try to keep these in sync, I'm just going to replace these with a single sentence intro and a link to the appropriate section in the dev guide.